src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/api/BundleController.kt

```diff
@@ -19,23 +19,20 @@ class BundleController(
             this(
                     context,
                     AppConfig(),
-                    OrderlyServerClient(AppConfig()).throwOnError()
+                    OrderlyServerClient(AppConfig())
```

Comment:
Don't really understand the changes to this file. Do we not need to specify throwOnError here because it's doing a pass through anyway?

Reply:
Sorry, yeah, exactly. In both the below methods the response from `orderly.server` is now passed along (including errors).